open-api/rest-catalog-open-api.yaml

```diff
@@ -1610,13 +1610,27 @@ components:
 
     PageToken:
       description:
-        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables). 
-        Clients will initiate the first paginated request by sending an empty `pageToken` e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`
-        signaling to the service that the response should be paginated.
+        An opaque token that allows clients to make use of pagination for list APIs
+        (e.g. ListTables). Clients may initiate the first paginated request by sending an empty
+        query parameter `pageToken` to the server e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`
+        
+        Servers that support pagination should identify the `pageToken` parameter and return a
+        `next-page-token` in the response if there are more results available.  After the initial
+        request, the value of `next-page-token` from each response must be used as the `pageToken`
+        parameter value for the next request. The server must omit or return `null` value for the
```

Comment:
A spec should not specify two options. I think the service should send `null`.

Reply:
Ok just want to clarify what we mean by the service should send `null` (does it mean the same as omitting the field). Since in the last line of the pagination description i think we wrote this

> Servers that do not support pagination should ignore the `pageToken` parameter and return
   all results in a single response. The `next-page-token` must be omitted or set to `null`

If setting `null` on service side means the same as service is omitting the `next-page-token` field in the response, something like this
```
ListIcebergNameResponse {
 "namespaces" : [...]
}
```
then would it clearer to just write the service should omit `next-page-token` when no more results are available?

Reply:
Sending `null` and omitting the field are different. This should explicitly send `null` for `next-page-token`. The idea is to always be clear about what, exactly, should be sent. Clients need to interpret both `null` and missing to stop further requests. But for a service what to send must always be clear.

Reply:
Thank you for the clarification will fix this.